### DISCUSSION TITLE
How do I backup my Teleport cluster to another server

### DISCUSSION THREAD

**Discussion by hakube** [gravitational/teleport#11827] [Q&A]
Just wanted to ask if this is possible. I am currently looking to backup my existing Teleport cluster from our datacentre and import it to a EC2 isntance as a backup access for customers when the Teleport server from our datacentre goes down.

I've tried the backup and restore process from the documentation but they didn't work.  Basically I was trying to do this to achieve redundancy to that my customers can not lose access if the Teleport server hosted on our datacentre goes down

**Comment by webvictim** [comment]
What didn't work about the instructions? What error do you get?
